spec/models/provider_user_spec.rb

```diff
@@ -78,36 +78,17 @@
     end
   end
 
-  describe '#can_view_safeguarding_information_for?' do
+  describe 'can_manage_organisations?' do
     let(:provider_user) { create :provider_user, :with_provider }
 
-    it 'is false without the correct permission' do
-      provider = provider_user.providers.first
-      expect(provider_user.can_view_safeguarding_information_for?(provider)).to be false
-    end
-
-    it 'is false without the correct permission for the given provider' do
-      provider_user.provider_permissions.update_all(view_safeguarding_information: true)
-      expect(provider_user.can_view_safeguarding_information_for?(build_stubbed(:provider))).to be false
-    end
-
-    it 'is true with the correct permission for the given provider' do
-      provider_user.provider_permissions.update_all(view_safeguarding_information: true)
-      provider = provider_user.providers.first
-      expect(provider_user.can_view_safeguarding_information_for?(provider)).to be true
+    it 'is false for users without the manage users permission' do
```

Comment:
Typo: `users` for `organisations`

Reply:
Updated here https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2231/files#diff-ee4451eba1a2bcdccf60d8f9bfea364cR84-R88